config/routes.rb

```diff
@@ -10,7 +10,7 @@
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
 
-  delete '/logout', to: 'sessions#destroy'
+  get '/logout', to: 'sessions#destroy'
```

Comment:
I don't remember what the page was, sadly. But yeah we can figure it out I'm sure. Just wanted to get the error fixed before checkin tomorrow.